common/src/app/common/colors.cljc

```diff
@@ -0,0 +1,15 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) UXBOX Labs SL
+
+(ns app.common.colors)
+
+(def black "#000000")
+(def canvas "#E8E9EA")
+(def default-layout "#DE4762")
+(def gray-20 "#B1B2B5")
+(def info "#59B9E2")
+(def color-test "#fabada")
```

Comment:
This can be just `test`

Reply:
I tried with test only but a it said test already exists.